aimmo-game/tests/test_simulation/test_pickups.py

```diff
@@ -44,18 +48,72 @@ def test_effect_added(self):
 class TestHealthPickup(_BaseCases.BasePickupTestCase):
     pickup_class = pickups.HealthPickup
 
+    def test_invulnerability_is_applied(self):
+        self.apply_pickup()
+        self.assertIsInstance(self.pickup, pickups.HealthPickup)
+
     def test_health_increases(self):
         self.apply_pickup()
         self.assertEqual(self.avatar.health, 8)
 
     def test_serialise(self):
```

Comment:
Asked @Spycho about this :)

Reply:
so is ok to remove? Or ...

Reply:
Don't think so. It's not a web service that we're running but this is sitting on flask so don't think we have pickups as django objects for example.

Some files have serialise() methods in them exactly because of this. Not 100% on this.

Reply:
ok, that's cool